build.sbt

```diff
@@ -252,10 +253,8 @@ lazy val http4s = project
       Seq(
         "dev.zio"                       %% "zio-interop-cats"              % zioInteropCats3Version,
         "org.typelevel"                 %% "cats-effect"                   % catsEffect3Version,
-        "org.http4s"                    %% "http4s-dsl"                    % http4sVersion,
-        "org.http4s"                    %% "http4s-server"                 % http4sVersion,
-        "org.http4s"                    %% "http4s-circe"                  % http4sVersion,
-        "io.circe"                      %% "circe-parser"                  % circeVersion,
+        "com.softwaremill.sttp.tapir"   %% "tapir-zio-http4s-server"       % tapirVersion,
+        "com.softwaremill.sttp.tapir"   %% "tapir-json-circe"              % tapirVersion,
```

Comment:
There isn't a `zio-json` tapir version is there?

Reply:
I was thinking to keep circe for the `Http4sAdapter` (to reduce changes for existing users), but now users are free to copy the 2-lines and use `TapirAdapter` and zio-json instead.

Reply:
An alternative way would be to keep caliban-http4s without any dependency to Json, but then all users have to import the right `tapir-json` lib and import it. What do you think is best?

Reply:
Is the json interface exposed the consumer? Any way we could make it configurable (i.e. implicit or via an explicit `JsonBlackbox`)?

Reply:
If you use `TapirAdapter`, it's configurable via an implicit `JsonCodec`. But in `Http4sAdapter`, I fixed it to Circe so that people who migrate have not much to do.